app/src/main/java/org/openobservatory/ooniprobe/test/test/AbstractTest.java

```diff
@@ -374,6 +374,24 @@ public boolean isAutoRun() {
         return Objects.equals(origin, AUTORUN);
     }
 
+	public static AbstractTest getTestByName(String name){
```

Comment:
I think the indenting of this function is a bit off: maybe a space vs tabs issue?

Reply:
Wasnt very visible in the IDE, will remove